ci/taos/webapp/pstree.php

```diff
@@ -31,7 +30,8 @@
  */
 
 echo "========================================================";
-$output = shell_exec("echo q | htop | aha --black --line-fix > htop.html");
+$cmd = "pstree -p | aha --black --line-fix";
```

Comment:
Replace htop with pstree to show a tree of processes for debugging.

Reply:
pstree does not show CPU / Memory usage, which is often useful. Do you intend to give up such information?

Reply:
The existing "PR: Resource Usage (top)" already shows  CPU and Memory usage. This PR is to show the running processes as a tree to monitor the relationship among the processes.

For example, "pstree -p'  is equal to 'htop + F5".